### ISSUE TITLE
Continuous game option at main menu

### ISSUE THREAD

**Issue by hytacker** [OpenRCT2/OpenRCT2#2214]
On the main menu of Orct you can select start a new game. Ater that the game shows you all of the parks you can play but al of these have objectives. I know you can make your own park with your own goal. And have the 'just have fun' option selected that has been added to the game. 

Would it be an idea to have the 'play a continuous game' option at the main menu or in the 'select a new game' menu. So that you don't have to alter an existing objective through the cheat code tab. And have this way of playing added to the base game in a clear and unique way.

**Comment by janisozaur** [comment]
"orct" can mean both "original" and "open", can you please use more verbose name?

**Comment by pizza2004** [comment]
I don't really understand this objective.  Can't you ignore the objective and just play the park anyway?  It doesn't close the park or force you to stop playing when you win/lose, does it, or am I wrong?

**Comment by HaasJona** [comment]
I agree with pizza2004, this suggestion doesn't make any sense to me. And yes, you can just continue to play regardless if you win or lose the objective.

**Comment by hytacker** [comment]
In some objective's the game forces you to build new rides at a certain rate. Otherwise your park rating will drop. If it drops below a certain point your park will be closed and the game is lost. This limits you in your way of creating a park. I've encounterd more situations in the game whereby the mechanics of the game hold you back. There's never been an option to play a game without a particular objective untill the 'just have fun' option came in. But this has to be selected specificly through the cheat menu. I'm trying to propose and option in which you can directly start a continouos game from the start. Instead of begining a campaign so to speak.

**Comment by pizza2004** [comment]
But when it closes the park can't you just reopen it and continue on like everything is fine?

**Comment by hytacker** [comment]
@pizza2004 No it can't.. It's the objetive that doesn't let you drop your park rating below 700. I always put alot of time in scenery. This is made impossible that way.

**Comment by pizza2004** [comment]
What you're asking for is sandbox mode, which I believe has already been a planned feature.  Also, I'm not really sure what you're talking about when you say it requires you to build new rides at specific intervals.  I'm pretty sure there are plenty of various ways to keep the park rating up.  If you want to build a lot of complex scenery without worrying about losing, it's possible to enable building while the game is paused.

**Comment by osconnoisseur** [comment]
I believe there is already a "Free Play" park with the have fun objective in the scenario menu. I think it's under the "other parks" sub menu.

**Comment by IntelOrca** [comment]
@osconnoisseur I have no idea what you are talking about.

**Comment by osconnoisseur** [comment]
Ah, never mind. I created a scenario for that purpose a long time ago and thought it was part of OpenRCT2. Disregard what I said.

Is our mapgen currently used at all in the game?

**Comment by PFCKrutonium** [comment]
Only if you use the scenario editor.
